specification/resources/resource-manager/readme.md

```diff
@@ -57,9 +57,18 @@ tag: package-managedapplications-2018-06
 ```
 
 ``` yaml $(package-deploymentscripts)
-tag: package-deploymentscripts-2019-10-preview
+tag: package-2020-01
```

Comment:
the tag naming is not consistent with previous one such as `package-subscription-2019-10-preview`, `package-resources-2019-10' etc.

Reply:
This was auto-generated for me by the OpenAPI Hub tool, but it looks like it dropped a new package tag in the middle of an existing one. If you can help me understand what these tags do and why they're added, I can try to straighten this out.

Reply:
I think I fixed it. Let me know if this looks better